cypress/elements/giftCard/giftCardList.js

```diff
@@ -1,3 +1,9 @@
 export const GIFT_CARD_LIST = {
-  issueCardButton: '[data-test-id="issueCardButton"]'
```

Comment:
Can we move this file to `cypress/elements/catalog/giftCards/giftCardList.js`  or smg like this?

Reply:
@karola312 giftCards are in the catalog section I mean that maybe we should create folder in the catalog and put there files with giftCardList.js. 
` cypress/elements/giftCards/giftCardList.js`  ->  `cypress/elements/catalog/giftCards/giftCardList.js `